.neutrinorc.js

```diff
@@ -34,5 +36,21 @@ module.exports = {
 			}
 		}],
 		'@neutrinojs/react-components',
+		(neutrino) => {
```

Comment:
It depends on your style, but you can optionally omit the parens here. :smile:

Reply:
Will change. I may also update the formatting while I'm at it... (stupid tabs)